tests/tck/features/steps/Steps.py

```diff
@@ -25,70 +27,97 @@ def step_impl(context):
 def step_impl(context):
     graphs.any_graph()
 
+@then(u'parameters are')
+def set_params(context):
+    global params
+
+    params = {}
+    for row in context.table.rows:
+        params[row[0]]=row[1]
+    print(params)
+
+@given(u'parameters are')
+def set_params(context):
+    global params
+
+    params = {}
+    for row in context.table.rows:
+        params[row[0]]=row[1]
+
 @given(u'having executed')
 def step_impl(context):
     global resultset
     global exception
+    global params
 
     exception = None
     query = context.text
     try:
-        resultset = graphs.query(query)
+        resultset = graphs.query(query, params)
     except Exception as error:
         resultset = None
         exception = error
+    params = None
```

Comment:
Isn't there a function which Behave calls after each Scenario, something similar to `teardown` in which we can reset our `params` ?

Reply:
yes